lib/travis/build/templates/header.sh

```diff
@@ -52,9 +53,12 @@ travis_cmd() {
   fi
 
   if [[ -n "$retry" ]]; then
-    travis_retry eval "$cmd"
+    travis_retry eval "$cmd $secure"
   else
-    eval "$cmd"
+    eval "$cmd $secure"
+    if [[ -n $secure && $? -ne 0 ]]; then
+      echo "The previous command failed"
```

Comment:
yes.  Maybe also a url to docs. Even if not a link, another place for further help.